spec/Section 3 -- Type System.md

```diff
@@ -248,6 +248,29 @@ a given GraphQL server expects. Any other input value, including float input
 values (such as `4.0`), must raise a query error indicating an incorrect type.
 
 
+#### Any
+
+The Any scalar type represents any value that can be represented by underlying
+serialization protocol (including lists and maps). It is intended to be an opt-out type in case when the
+exact type of possible data is not known in advance or multiple types should be
+supported.
+
+Note: {null} value supplied as an argument or field value should be interpreted
```

Comment:
This note feels out of place, as this statement is true for all scalar types, not just this new Any type. However I do think clarification is needed about how Any interacts with Non-Null.